src/components/Autocomplete.jsx

```diff
@@ -20,17 +20,31 @@ const Autocomplete = ({ id, suggest, label, ...rest }) => {
           {label}
         </label>
       )}
-      <UkAutocomplete
-        id={id}
-        className="dg_form-field_input--text"
-        source={handleSource}
-        showNoOptionsFound={false}
-        templates={{
-          inputValue: FormatAddress,
-          suggestion: FormatAddress,
-        }}
-        {...rest}
-      />
+      <div style={{ position: "relative" }}>
+        <UkAutocomplete
+          id={id}
+          className="dg_form-field_input--text"
+          source={handleSource}
+          showNoOptionsFound={false}
+          templates={{
+            inputValue: FormatAddress,
+            suggestion: FormatAddress,
+          }}
+          {...rest}
+        />
+        <button
+          type="submit"
+          style={{
+            position: "absolute",
```

Comment:
this needs to be moved to css as it responds based on screen size.

Reply:
this probably should be an addition to the dotgov-components project. going to talk to web servcies about catching this work after the 27th.